tests/getRatings.test.js

```diff
@@ -0,0 +1,373 @@
+// import { describe, test, expect } from "@jest/globals;";
+import getRecentRatings from "./getRecentRatings";
+
+describe("getAverageRatings()", () => {
+  test("includes a review with 5 rating", async () => {
+    const data = await getRecentRatings({
+      loadAllRatings: async () => {
+        [
```

Comment:
I think maybe this code doesn't do what you expect it to do. "Fat arrow functions" can have two forms.

The short form is a single statement and returns the value of the expression to the right of `=>`, e.g.:

```js
const x = () => 5; // Returns 5
const y = () => [1, 2, 3]; // Returns array
const z = () => x(); // Returns the return value of x, i.e. 5
```

The other form contains a block of statements, can contain more than a single statement, and doesn't return anything automatically. This is achieved by putting curly braces (`{}`) after the fat arrow:

```js
const x = () => {
  console.log('First statement');
  const value = 2 + 3;
  return value; // returns 5
};

const y = () => {
  console.log('Prints something');
  // Does not return anything
};
```

Your code is of type two, i.e. you are creating a function that doesn't return anything, because you have a curly brace after the fat arrow.

I think you want your function to return an object, but the problem is that the object literal uses curly braces as well. So if you want to use the first version to return an object, you need to wrap that object in parentheses:

```
const data = await getRecentRatings({
  loadAllRatings: async () => ({ // <-- note the parenthesis that wraps the curly brace
    // object contents go here
  }),
});
```